kview/src/main/java/dev/ikm/komet/kview/mvvm/view/genediting/PropertiesController.java

```diff
@@ -105,25 +107,23 @@ private void setupShowingPanelHandlers() {
 
         editFieldsJfxNode = FXMLMvvmLoader.make(config);
 
-        Config closePropertiesConfig = new Config(this.getClass().getResource("close-properties.fxml"))
-                .addNamedViewModel(new NamedVm("genEditingViewModel", genEditingViewModel));
-        JFXNode<Pane, ClosePropertiesController> closePropsJfxNode = FXMLMvvmLoader.make(closePropertiesConfig);
+        ConfirmationPaneCommonViewModel confirmationViewModel = new ConfirmationPaneCommonViewModel();
+
+        Config closePropertiesConfig = new Config(this.getClass().getResource("/dev/ikm/komet/kview/mvvm/view/confirmation/confirmation-pane-common.fxml"))
```

Comment:
Can the common fxml file be in the same namespace as `ConfirmationPaneCommonController`?
**Reason:** That way the it could look like:
```java
Config closePropertiesConfig = new Config(ConfirmationPaneCommonController.class.getResource("confirmation-pane-common.fxml"));
```

Naming is hard, but I'd prefer the controller to be named: `ConfirmationPaneController`.